superset/reports/commands/execute.py

```diff
@@ -408,9 +408,11 @@ def _send(
             try:
                 if app.config["ALERT_REPORTS_NOTIFICATION_DRY_RUN"]:
                     logger.info(
-                        "Would send notification for alert %s, to %s",
+                        "Would send notification for alert %s, to %s. "
+                        "%s is enabled, set it to False to send notifications.",
                         self._report_schedule.name,
                         recipient.recipient_config_json,
+                        app.config["ALERT_REPORTS_NOTIFICATION_DRY_RUN"],
```

Comment:
Would this not produce `True is enabled, set it to False to send notifications.`, as that's the value of the config param, not it's name?

Reply:
Yes, I thought I pushed a fix for that 😞

Reply:
... I pushed to the wrong remote. Fixed